azure-pipelines.yml

```diff
@@ -13,11 +13,13 @@ variables:
   imageNameBgMailer: 'teacher-training-bg-mailer'
   dockerOverride: 'docker-compose -f docker-compose.yml'
 
+
 steps:
 - script: |
     set -x
     GIT_SHORT_SHA=$(echo $(Build.SourceVersion) | cut -c 1-7)
     docker_path=$(dockerHubUsername)/$(imageName)
+    docker_middleman_path=$(dockerHubUsername)/$(imageName)-middleman
```

Comment:
calling it `middleman`

seems weird...
may `doc-gen` or something along those lines to reflect that that docker file is do....

run  middleman to generate static documentation